tests/unit/rpc/handlers/BookChangesTests.cpp

```diff
@@ -222,3 +223,51 @@ TEST_F(RPCBookChangesHandlerTest, NormalPath)
         EXPECT_EQ(*output.result, json::parse(kEXPECTED_OUT));
     });
 }
+
+TEST_F(RPCBookChangesHandlerTest, NormalPathWithDomain)
+{
+    static constexpr auto kEXPECTED_OUT =
+        R"JSON({
+            "type":"bookChanges",
+            "ledger_hash":"4BC50C9B0D8515D3EAAE1E74B29A95804346C491EE1A95BF25E4AAB854A6A652",
+            "ledger_index":30,
+            "ledger_time":0,
+            "validated":true,
+            "changes":[
+                {
+                    "currency_a":"XRP_drops",
+                    "currency_b":"rK9DrarGKnVEo2nYp5MfVRXRYf5yRX3mwD/0158415500000000C1F76FF6ECB0BAC600000000",
+                    "volume_a":"2",
+                    "volume_b":"2",
+                    "high":"-1",
+                    "low":"-1",
+                    "open":"-1",
+                    "close":"-1",
+                    "domain":"F10D0CC9A0F9A3CBF585B80BE09A186483668FDBDD39AA7E3370F3649CE134E5"
+                }
+            ]
+        })JSON";
+
+    EXPECT_CALL(*backend_, fetchLedgerBySequence).Times(1);
```

Comment:
nit: Times(1) is default (here and below)

Reply:
Improved in the whole file